package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Firefox Accounts Profile service.",
   "scripts": {
     "outdated": "npm outdated --depth 0",
```

Comment:
can this be updated to use `yarn outdated`?  I don't think you need the `--depth` part